src/NServiceBus.Transport.RabbitMQ/NServiceBus.Transport.RabbitMQ.csproj

```diff
@@ -12,14 +12,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
     <Reference Include="System.Configuration" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Fody" Version="2.1.2" PrivateAssets="All" />
```

Comment:
Yes, we were getting it transitively before, but that means we were picking up an older version since transitive depedenciies resolve to the lowest version allowed in the version range.

Since neither of those packages have been updated to point to a newer version, we need the top-level dependency to pick up the fixes in the latest Fody, which include the Mono.Cecil fixes needed for #387.